prog/dftb+/lib_solvation/lebedev.F90

```diff
@@ -0,0 +1,5541 @@
+!--------------------------------------------------------------------------------------------------!
+!  DFTB+: general package for performing fast atomistic simulations                                !
+!  Copyright (C) 2006 - 2020  DFTB+ developers group                                               !
+!                                                                                                  !
+!  See the LICENSE file for terms of usage and distribution.                                       !
+!--------------------------------------------------------------------------------------------------!
+
+#:include 'common.fypp'
+
+!> Generator for Lebedev-Laikov grids.
+!>
+!> V.I. Lebedev, and D.N. Laikov, "A quadrature formula for the sphere of the
```

Comment:
Is this a fresh implementation from this paper, or is there additional history for these routines? Looks like it might be an offshoot of either the [LGPL routines](https://people.sc.fsu.edu/~jburkardt/f_src/sphere_lebedev_rule/sphere_lebedev_rule.f90) or the van Wuellen [Fortran port](http://www.ccl.net/cca/software/SOURCES/FORTRAN/Lebedev-Laikov-Grids/Lebedev-Laikov.F) of the original C, which carries a requirement on including the reference in publications and manuals (not sure at which stage the jburkardt implementation removed/re-implemented this).

Reply:
Yes, you are right, I will reference it properly.